mypy/message_registry.py

```diff
@@ -74,6 +74,8 @@
 FUNCTION_TYPE_EXPECTED = "Function is missing a type annotation"  # type: Final
 ONLY_CLASS_APPLICATION = "Type application is only supported for generic classes"  # type: Final
 RETURN_TYPE_EXPECTED = "Function is missing a return type annotation"  # type: Final
+RETURN_TYPE_EXPECTED_DISALLOW_UNTYPED = \
+    "Function must have a return type annotation ('disallow_untyped_defs' enabled)"  # type: Final
```

Comment:
Although it is mentioned in the issue, I would not mention the name of option of the flag in the error message. It may easily get out of sync. So I would remove the part in `(...)`.